### PR TITLE
add section _Reviewing and validating pull request_

### PR DESCRIPTION
**Pull request by edouard-lopez**

### CODE REVIEW

File: git-workflow.md [hunk 1]
```diff
@@ -7,6 +7,20 @@
...
+
+### How to do it?
+
+* set JIRA ticket to `in review`
+* assign to someone, self-assign or add _:pray:`please review`_
+* review, at least one, by someone who don't write the code
+  * fetch PR locally with 
+
+		git pull-request --download <PR-number>
+* [coming soon] if valid add `mergeit` label to trigger zuul
```
Comment (sileht) [inline git-workflow.md]: We can remove `coming soon`, the steps looks good.

Comment (sileht) [reply git-workflow.md]: In review tab you also need to push "Approve" Button.

File: git-workflow.md [hunk 2]
```diff
@@ -7,6 +7,22 @@
...
+
+* set JIRA ticket to `in review`
+* assign to someone, self-assign or add _:pray:`please review`_
+* review, at least one, by someone who don't write the code
+  * fetch PR locally with 
+
+		git pull-request --download <PR-number>
+
+* Once PR is valid, "Approve" it (in review tab)
+* if valid add `mergeit` label to trigger zuul
```
Comment (fblackburn1) [inline git-workflow.md:22]: What is the difference to add the label `mergeit` when the ticket is ready to be reviewed VS when the ticket is reviewed? In all case, zuul will wait the approval of someone before merging, no?

Comment (sileht) [reply git-workflow.md:22]: > ... zuul will wait the approval of someone before merging ...

Unfortunately, this is a bad implementation of zuul+github, on Gerrit you don't have such restriction once you put `mergeit` (called `Approved` in gerrit) and tests passed, the change is merged.

This `mergeit` + `at least 1 approvals` doesn't looks configurable for now :(

Comment (fblackburn1) [reply git-workflow.md:22]: Oh good to know, thanks

**Review by fblackburn1 — APPROVED** [review]

**Review by sduthil — APPROVED** [review]

**Pull request merged**